packages/serialization/json/test/common/jsonSerializationWriter.ts

```diff
@@ -78,6 +78,23 @@ describe("JsonParseNode", () => {
     const result = JSON.parse(contentAsStr);
```

Comment:
we probably also need a unit test with the backing store and to correct the implementation divergence with dotnet.
https://github.com/microsoft/kiota-typescript/issues/1079#issuecomment-1959515442